distribution/systemd/vector.service

```diff
@@ -11,6 +11,7 @@ ExecStart=/usr/bin/vector
 ExecReload=/bin/kill -HUP $MAINPID
 Restart=no
 AmbientCapabilities=CAP_NET_BIND_SERVICE
+EnvironmentFile=/etc/default/vector
```

Comment:
Not a big deal since we are packaging one, but I've usually seen these as `EnvironmentFile=-` to avoid erroring if the file doesn't exist.